src/jit/morph.cpp

```diff
@@ -2564,6 +2602,7 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
     regMaskTP       argSkippedRegMask    = RBM_NONE;
 
 #if defined(_TARGET_ARM_) || defined(_TARGET_AMD64_)
+    // Note this in ONLY used by _TARGET_ARM_
     regMaskTP       fltArgSkippedRegMask = RBM_NONE;
```

Comment:
in=>is. But why put it under `_TARGET_AMD64_` if it's not used there, instead of deleting that part of the #ifdef.